README.md

```diff
@@ -286,6 +106,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
- [ ] Kindly add a License file to the root of your project to stop it from breaking.
- Including a license file in a README is important as it clarifies the terms and conditions under which the software can be used, providing legal protection and promoting open collaboration while preventing potential misuse or infringement.